src/main.cpp

```diff
@@ -2700,14 +2714,14 @@ bool Session::InitializeDRM()
         AP4_FormatHex(reinterpret_cast<const AP4_UI08*>(defkid), 16, hexkid), hexkid[32] = 0;
         kodi::Log(ADDON_LOG_DEBUG, "Initializing stream with KID: %s", hexkid);
 
-        for (unsigned int i(1); i < ses; ++i)
+        /*for (unsigned int i(1); i < ses; ++i)
```

Comment:
its oversight or why has been commented out?

Reply:
I totally forgot about this... I need to do my own review before submitting!

This needs some thought actually, I will try to find a good solution. But in any case, CBC can't be done with a single CDM session as there are separate patterns required between audio and video.

Reply:
The main problem being if we have an mpd with say 10 video representations, each with their own key then license server get contacted 10 times == slow to start stream. This would already be affecting Android with cenc streams btw